lib/hammer_cli_foreman/smart_class_parameter.rb

```diff
@@ -160,9 +160,16 @@ class AddOverrideValueCommand < HammerCLIForeman::CreateCommand
       end
     end
 
-    class RemoveOverrideValueCommand < HammerCLIForeman::DeleteCommand
+    HammerCLIForeman::SmartClassParameter.lazy_subcommand('add-override-value', _("Create an override value for a specific smart variable"),
+      'HammerCLIForeman::SmartClassParameter', 'hammer_cli_foreman/smart_class_parameter',
```

Comment:
This needs to be `HammerCLIForeman::SmartClassParameter::AddMatcherCommand` otherwise `hammer sc-param add-override-value -h` prints help for `sc-param` not `add-override-value`.